### PR TITLE
Spring Boot Starter Wallet

### PR DESCRIPTION
**Pull request by andytael**
Initial commit of Spring Boot Starter Wallet (2.7.7 and 3.0.2)

### CODE REVIEW

File: spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/oracle-spring-boot-starter-wallet-2.7.7.pom [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <version>2.7.7</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>oracle-spring-boot-starter-wallet</artifactId>
+    <version>2.7.7</version>
+    <packaging>pom</packaging>
+
+    <name>Oracle Spring Boot Starter - Wallet</name>
+    <description>Oracle's implementation of Spring Boot Starter for using with Oracle UCP</description>
```
Comment (markxnelson) [inline spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/oracle-spring-boot-starter-wallet-2.7.7.pom:21]: UCP - should be wallet

File: spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        <version>2.7.7</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>oracle-spring-boot-starter-wallet</artifactId>
+    <version>2.7.7</version>
+    <packaging>pom</packaging>
+
+    <name>Oracle Spring Boot Starter - Wallet</name>
+    <description>Oracle's implementation of Spring Boot Starter for using with Oracle UCP</description>
```
Comment (markxnelson) [inline spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/pom.xml:21]: ucp should be wallet

**Review by markxnelson — CHANGES_REQUESTED** [review]
couple of small changes, otherwise LGTM

File: spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/pom.xml (hunk 2 above)
Comment (andytael) [reply spring/oracle-spring-boot-starters/2.7.7/oracle-spring-boot-starter-wallet/pom.xml]: Fixes are in!

**Review by markxnelson — APPROVED** [review]
LGTM

**Pull request merged**